.github/workflows/docker.yml

```diff
@@ -50,6 +50,16 @@ jobs:
         shell: pwsh
         name: "Get Kiota's version-number from .csproj"
 
+      - run: ./scripts/get-release-notes.ps1 -version "${{ steps.getversion.outputs.version }}" -check
+        if: ${{!contains(github.ref, env.PREVIEW_BRANCH)}}
+        name: "Get release notes from CHANGELOG.md"
+        shell: pwsh
+
+      - run: ./scripts/get-release-notes.ps1 -version Unreleased -check
```

Comment:
Instead of `check`, it would read a lot better to call it something like `createReleaseNotes` and change the condition where it is checked in get-release-notes.ps1 . The `check` switch isn't a clear parameter name for what this does. Applies to both steps.

In get-release-notes.ps1, consider:
* Change  `check` parameter name to  `createReleaseNotes`
* Change condition `($check -eq $false)` to `($createReleaseNotes -eq $true)`

In docker.yml, then remove the `-check` switch.

In ci-build.yml, update the 4 scenarios that run get-release-notes.ps1 to reflect this change.